docs/tutorials/send_receive.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Sending & receiving message over a deployed broker"  
+description: "Steps to get a producer and a consummer exchanging messages over a deployed broker on OpenShift"
+draft: false
+images: []
+menu:
+  docs:
+    parent: "tutorials"
+weight: 110
+toc: true
+---
+
+### Prerequisite
+
+Before you start, you need to have access to a running Kubernetes cluster
+environment. A [Minikube](https://minikube.sigs.k8s.io/docs/start/) instance
+running on your laptop will do fine.
+
+### Step 1 - Deploy the operator
+
+#### create the namespace
+
+```shell script
+$ kubectl create namespace activemq-artemis-operator
+```
+
+Go to the root of the operator repo and install it:
+
+```shell script
+$ cd activemq-artemis-operator
+$ ./deploy/install_opr.sh
+```
+
+Deploy the Operator:
+
+```shell script
+$ kubectl create -f deploy/operator.yaml --namespace activemq-artemis-operator
+```
+
+Wait for the Operator to start (status: `running`).
+```shell script
+$ kubectl get pod --namespace activemq-artemis-operator
+```
+
+### Step 2 - Deploying the ActiveMQ Artemis Broker
+
+For this tutorial we need to:
+
+* have a broker that is able to listen to any network interface. For that we
+  setup an `acceptor` that will be listening on every interfaces on port
+  `62626`.
+* have queues to exchange messages on. These are configured by the broker
+  properties. Two queues are setup, one called `APP.JOBS` that is of type
+  `ANYCAST` and one called `APP.COMMANDS` that is of type `MULTICAST`.
+
+```shell script
+$ kubectl apply -f - <<EOF
+apiVersion: broker.amq.io/v1beta1
+kind: ActiveMQArtemis
+metadata:
+  name: default
+  namespace: activemq-artemis-operator
+spec:
+  acceptors:
+    - bindToAllInterfaces: true
+      name: acceptAll
+      port: 62626
+  brokerProperties:
+    - addressConfigurations."APP.JOBS".routingTypes=ANYCAST
+    - addressConfigurations."APP.JOBS".queueConfigs."APP.JOBS".routingType=ANYCAST
+    - addressConfigurations."APP.COMMANDS".routingTypes=MULTICAST
+  deploymentPlan:
+    image: 'quay.io/artemiscloud/activemq-artemis-broker-kubernetes:latest'
+    initImage: 'quay.io/artemiscloud/activemq-artemis-broker-init:latest'
+    requireLogin: false
+    size: 1
```

Comment:
Those deploymentPlan settings could be removed because they match the default values

Reply:
What conditions are failing in your ActiveMQArtemis CR?

Reply:
After trying to reproduce, and executing 

```
kubectl apply -f - <<EOF
apiVersion: broker.amq.io/v1beta1
kind: ActiveMQArtemis
metadata:
  name: send-receive
  namespace: send-receive-project
spec:
  ingressDomain: localresolvconf.com
  acceptors:
    - name: sslacceptor
      port: 62626
      expose: true
      sslEnabled: true
      sslSecret: send-receive-sslacceptor-secret
  brokerProperties:
    - addressConfigurations."APP.JOBS".routingTypes=ANYCAST
    - addressConfigurations."APP.JOBS".queueConfigs."APP.JOBS".routingType=ANYCAST
    - addressConfigurations."APP.COMMANDS".routingTypes=MULTICAST
EOF
```
it worked fine, so I don't know what happened the last time.